winmake.ps1

```diff
@@ -44,6 +44,14 @@ function Make-Clean{
     }
 }
 
+function Local-Unit {
+    Build-Ginkgo
+    $skippackages="hack,internal\domain\infra\abi,internal\domain\infra\tunnel,libpod\lock\shm,pkg\api\handlers\libpod,pkg\api\handlers\utils,pkg\bindings,"
+    $skippackages+="pkg\domain\infra\abi,pkg\emulation,pkg\machine\apple,pkg\machine\applehv,pkg\machine\e2e,pkg\machine\libkrun,"
+    $skippackages+="pkg\machine\provider,pkg\machine\proxyenv,pkg\machine\qemu,pkg\specgen\generate,pkg\systemd,test\e2e,test\utils"
+    Run-Command "./test/tools/build/ginkgo.exe -vv -r --tags `"$remotetags`" -timeout=90m --trace --no-color --skip-package `"$skippackages`""
```

Comment:
does this need to have two dashes for timeout?
Also 90m is to high for unit tests, the default task timeout is 20m so it should be lower than that. Or you just skip setting a timeout.

Reply:
Thank you, I have fixed it and set the timeout to 15 minutes. Previous run took around 7 min.